curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/arguments-optional.md

```diff
@@ -24,28 +24,30 @@ If either argument isn't a valid number, return undefined.
 
 # --hints--
 
-`addTogether(2, 3)` should return 5.
+`addTogether("https://www.youtube.com/watch?v=dQw4w9WgXcQ")` should return `undefined`.
 
 ```js
-assert.deepEqual(addTogether(2, 3), 5);
+assert.isUndefined(addTogether('https://www.youtube.com/watch?v=dQw4w9WgXcQ'));
 ```
 
-`addTogether(23, 30)` should return 53.
+`addTogether("2", 3)` should return `undefined`.
 
 ```js
-assert.deepEqual(addTogether(23, 30), 53);
+assert.isUndefined(addTogether('2', 3));
 ```
 
-`addTogether(5)(7)` should return 12.
+`addTogether(5)` should return another function.
 
 ```js
-assert.deepEqual(addTogether(5)(7), 12);
+assert.deepEqual(typeof(addTogether(5)), 'function');
 ```
 
-`addTogether("https://www.youtube.com/watch?v=dQw4w9WgXcQ")` should return `undefined`.
+
+`addTogether(5)` should return another function then return sum if the other function has a parameter. should return 12.
```

Comment:
Also, this isn't the wording requested.

Please use the ordering and text given in the issue.